### PR TITLE
docs(ee/config) complete property reference for enterprise TD-39

### PR DESCRIPTION
**Pull request by h3xar0n**
### Summary

Provides a complete property reference for Kong Enterprise; most sections outside of Kong Manager and Dev Portal were missing. Any shared properties with Core have links to the relevant doc.
 
### Full changelog

* Add either a description or a link to all missing properties in Enterprise
* Replace sub-section in Audit Log with a link to the main property reference
* Fix any noticeable formatting issues

### Issues resolved

Fix TD-39 (internal)

### Checklist:
- [x] [Commit message & atomicity](https://github.com/Kong/docs.konghq.com/blob/master/CONTRIBUTING.md#commit-atomicity) checked
- [x] Documentation
- [x] Spellchecked my updates
- [x] Ready to be merged 
  _Needed in order to merge_:
  - [x] Although we have an entry for `admin_invite_email`, it's nowhere in `kong.conf` and may need to be removed.
  - [x] Write all of the properties instead of linking to Kong reference. This will avoid confusion and improve the reader experience.
  - [x] Remove the ToC for now and create a manual ToC with general sections; this will all be replaced with a left-nav

### CODE REVIEW

File: app/enterprise/0.34-x/property-reference.md [hunk 1]
```diff
@@ -2,27 +2,63 @@
 title: Configuration Property Reference for Kong Enterprise
 ---
 
-## Kong Manager
+## General
```
Comment (levfishbluefish) [inline app/enterprise/0.34-x/property-reference.md]: Could we include something here to explain why the first section is just links to another page? It does seem odd to just say "see Kong's configuration reference" as users might be like, "I thought I was!' IDK something like  - 'properties in this section are in both Kong and Kong Enterprise. Please see the Kong configuration file for further information"

**Comment by levfishbluefish** [comment]
There are a LOT of duplicate properties, I am wondering if it would be easier to treat Kong and Kong Enterprise docs as totally separate docs (aka Enterprise customers should never have to view Kong and vice versa) and just include all the property info here as well so Enterprise users don't have to use two different docs

**Comment by h3xar0n** [comment]
> There are a LOT of duplicate properties, I am wondering if it would be easier to treat Kong and Kong Enterprise docs as totally separate docs (aka Enterprise customers should never have to view Kong and vice versa) and just include all the property info here as well so Enterprise users don't have to use two different docs

That is definitely a better reader experience. To maintain it, we'll just need to be sure to diff the kong.conf every single version.

**Comment by h3xar0n** [comment]
> Found an existing typo - line 822 - 'port_gui_use_subdomains' should be 'portal_gui_use_subdomains'

Since it may take longer to merge this one, I've made a separate PR here:
https://github.com/Kong/docs.konghq.com/pull/1099

**Pull request merged**